### PR TITLE
Update SSSOM Logo in docs

### PR DESCRIPTION
**Pull request by matentzn**
The SSSOM Logo has had a typo for all of its life: ontology instead of "ontological" to express the fact that this is not just about "mappings across ontologies", but "semantic" mappings in general. @sagehrke has updated the logos to that end - big thanks 🎉 

- [X] `docs/` have been added/updated if necessary
- [ ] `make test` has been run locally
- [ ] tests have been added/updated (if applicable)
- [ ] [CHANGELOG.md](https://github.com/mapping-commons/sssom/blob/master/CHANGELOG.md) has been updated.

### CODE REVIEW

**Comment by gouttegd** [comment]
So, should the standard really have been called “Simple Standard for Sharing **Semantic** Mappings” (SSSSM)? And here I thought that SSSOM was difficult to pronounce… :D

**Review by gouttegd — APPROVED** [review]
No objection, but is there a reason why we don’t commit the logo file into _this_ repository?

Having the logo hosted elsewhere means it can disappear anytime for reasons that are beyond our control. In fact, this is exactly what has already happened: the logo is currently not visible on the [public website](https://mapping-commons.github.io/sssom/spec/), because the repository hosting it has changed.

Having the logo stored within _this_ repository would prevent that from happening again.

**Comment by matentzn** [comment]
> So, should the standard really have been called “Simple Standard for Sharing Semantic Mappings” (SSSSM)? And here I thought that SSSOM was difficult to pronounce… :D

You are laughing now but I actually suggested this. Some people in Monarch were already mad we had settled from SSSOM, being the "worst of all possible acronyms", "terrible", "how could you", but when I came to them and suggested "SSSSM" and to pronounce it something like "sim" they blocked me with very clear words. The schema extension could even be called `SSSSS` (the last S being `Schema Mappings`) but.. ok. We already punished the world good enough with SSSOM!

**Comment by gouttegd** [comment]
> when I came to them and suggested "SSSSM" and to pronounce it something like "sim" they blocked me with very clear words.

Pity. They should have recognized that at least you didn’t come up with “SSSSMS” (Simple Standard for Sharing Semantic Mapping **Sets**). :D

**Comment by matentzn** [comment]
Haha. I updated committed the logo and updated the references. Thanks for the suggestion!

**Comment by gouttegd** [comment]
Thanks, but without any precision anywhere, the file in `src/docs/images/sssom-banner.png` will be assumed to be published under the same terms as the rest of the repository (i.e. the 3BSD license in the `LICENSE` file), which are _not_ the terms under which it is published [on the original repository](https://github.com/tis-lab/closed-illustrations/blob/master/LICENSE.md).

We need either to make sure that the illustrator is happy with the terms change, or add a notice somewhere in the README to state that the logo file is _not_ covered by the terms set forth in LICENSE.

**Comment by jmcmurry** [comment]
Use of all logos in this repo is blanket permitted (without written permission) by members of the internal Monarch team and all collaborators on Monarch flagship products

File: src/docs/spec.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Simple Standard for Sharing Ontological Mappings (SSSOM)
 
-<img src="https://github.com/jmcmurry/closed-illustrations/raw/master/logos/sssom-logos/sssom_logo_black-banner.png" />
-
+![SSSOM banner](../images/sssom-banner.png)
```
Comment (ehartley) [inline src/docs/spec.md:3]:
```suggestion
![SSSOM banner](images/sssom-banner.png)
```
The images folder is in the same directory as spec.md.

**Review by ehartley — APPROVED** [review]
Everything looks good!

**Comment by matentzn** [comment]
@gouttegd Do you approve as well?

**Review by gouttegd — APPROVED** [review]
Sure.

**Pull request merged**